src/main/groovy/org/transmartproject/batch/tag/InvalidInput.groovy

```diff
@@ -0,0 +1,7 @@
+package org.transmartproject.batch.tag
+
+/**
+ * Invalid input.
+ */
+class InvalidInput extends RuntimeException {
```

Comment:
Exceptions conventionally end in `Exception`. Adding this exception also seems unnecessary. See comment on `setShownIfEmpty`.